var/spack/repos/builtin/packages/intel-oneapi-mkl/package.py

```diff
@@ -220,9 +220,14 @@ def _find_mkl_libs(self, shared):
         resolved_libs = find_libraries(libs, lib_path, shared=shared)
         # Add MPI libraries for cluster support. If MPI is not in the
         # spec, then MKL is externally installed and application must
-        # link with MPI libaries
-        if self.spec.satisfies("+cluster ^mpi"):
-            resolved_libs = resolved_libs + self.spec["mpi"].libs
+        # link with MPI libaries. If MPI is in spect, but there are no
```

Comment:
Minor nit:  Do you mean `MPI is in spec`?

Reply:
Fixed. Thanks. Can you re-approve? I have not heard back from @lorisercole but I did some testing on my own.